README.md

```diff
@@ -1 +1,44 @@
-# Hello-Microverse
\ No newline at end of file
+
+# Hello Microverse
+
+> Setting up a Hello Microverse.
+![screenshot](./photos/shot.png)
```

Comment:
- [ ] [Optional] Great work adding a screenshot for your project, Next time please add section `## Screenshot` after the project description.